pkg/apis/lakom/register.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package lakom
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "lakom.extensions.config.gardener.cloud"
```

Comment:
```suggestion
const GroupName = "lakom.extensions.gardener.cloud"
```

Please, update all other occurrences where the group name with `.config.` used.